### PR TITLE
Restyling & small enhancements

### PR DESCRIPTION
**Pull request by djowvani**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 #theme: "hallios"
 #remote_theme: "hallios"
-minimal_mistakes_skin: "dark" # "air", "aqua", "contrast", "dark", "dirt", "neon", "mint", "plum", "sunrise"
+minimal_mistakes_skin: "hallios1" # "air", "aqua", "contrast", "dark", "dirt", "neon", "mint", "plum", "sunrise", "hallios1", "hallios2", "hallios3"
```
Comment (djowvani) [inline _config.yml:16]: We have 3 theme options here:

1 - hallios colors
2 - the most "comfort" one considering design best practices
3 - the first one + additional styling on content containers

feel free to experiment and tweak them to your desire, in opinion the color red was a little hard to fit nicely.

File: _includes/footer.html [hunk 2]
```diff
@@ -30,4 +30,5 @@
   site.data.ui-text[site.locale].powered_by | default: "Powered by" }}
   <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp;
   <a href="https://garrettstimpson.ca" rel="dofollow">gary23w</a>.
+  <a href="https://github.com/djowvani" rel="dofollow">djowvani_</a>.
```
Comment (djowvani) [inline _includes/footer.html:33]: I don't mind if you folks prefer to remove this, after all it was just a help

File: _sass/hallios/_masthead.scss [hunk 3]
```diff
@@ -4,12 +4,20 @@
...
   z-index: 20;
+  @if variable-exists(border-color) {
+    border-bottom: 1px solid $border-color;
+  }
+  @if variable-exists(header-border-color) {
+    border-bottom: $header-border-color;
+  }
+  @if variable-exists(header-background-color) {
+    background-color: $header-background-color;
+  }
```
Comment (djowvani) [inline _sass/hallios/_masthead.scss:20]: These ifs ensure that even if the theme doesn't have the variable, it won't crash the app

**Comment by t3km4n** [comment]
Merging PR

**Pull request merged**